mkdocs_static_i18n/plugin.py

```diff
@@ -149,6 +153,16 @@ def _set_languages_options(self, config):
             self.default_language_options["site_name"] = default_language_config.get(
                 "site_name", config["site_name"]
             )
+            # Check if default homepage is set as extra in configuration
```

Comment:
This is only true for `material` if I'm not mistaken? mention it on the comment please